app-core-test/src/test/java/net/spals/appbuilder/app/core/jaxrs/SampleJaxRsWebAppFTest.java

```diff
@@ -195,10 +197,21 @@ public void testModelInjection() {
     }
 
     @Test
+    public void testMonitorInjection() {
+        final Injector serviceInjector = sampleApp.getServiceInjector();
+        assertThat(serviceInjector.getInstance(Tracer.class), instanceOf(NoopTracer.class));
```

Comment:
A practice I've been using lately is calling unit tests
// arrange

// act

// assert
I don't specifically make those comments but I arrange the code as such.

So
 final Injector serviceInjector = sampleApp.getServiceInjector();
 \\ new line to purposely show separation from arrange and act, pulling actual value into its own call
 final Tracer tracer = serviceInjector.getInstance(Tracer.class);
\\ new line again to show separation of act and assert
 assertThat(tracer, instanceOf(NoopTracer.class));

Reply:
I like that model. I think I've heard something similar before, but obviously I'm not disciplined about it. I'll start using that.